### ISSUE TITLE
& is displayed as &amp;

### ISSUE THREAD

**Issue by r0747249** [plexinc/plex-media-player#980]
### Test environment

**PMS Version:** 1.20.1.3213
**PMP Version:** 1.17.0.1376-439f8b7f
**Platform:** Windows

### Current behavior
In the movie collection summaries `&` is displayed as `&amp;`. This happens in the Plex for Windows app and in the Plex-Web but not in the IOS app. 

**Proof:**
![Plex for Windows app](https://user-images.githubusercontent.com/43750381/91070566-2e252f00-e637-11ea-9a24-e8e9b3424de7.jpg)
![IOS-app](https://user-images.githubusercontent.com/43750381/91071226-3893f880-e638-11ea-817c-1ce3ec0a78e1.jpg)

**Comment by mseeley** [comment]
Am I correct that you see the same behavior when viewing that collection using Plex Web on http://app.plex.tv/?

Also, the Slack notification we received from this is a good demonstration of an opposite issue. Text encoding is hard. ;)

![image](https://user-images.githubusercontent.com/30019/91087427-1e342d80-e5ec-11ea-8109-ee5d0360aeb9.png)

**Comment by r0747249** [comment]
Yes it also show up in the plex-web app. And I don't know if I should open another thread about this but I also can't lock those summaries because the lock button is missing
![image](https://user-images.githubusercontent.com/43750381/91100826-26c94a00-e666-11ea-86fd-135838f95b77.png)
